src/pages/index.mdx

```diff
@@ -12,17 +9,360 @@ export const disableEditGithub = true
+provider: nitric/azure@1.1.1
+region: East US
+```
+
+```yaml {{ label:"nitric.prod.yaml", title: "Google Pulumi" }}
+provider: nitric/gcp@1.1.1
+region: us-east1
+```
+
+```yaml {{ label:"nitric.prod.yaml", title: "AWS Terraform" }}
+provider: nitric/awstf@1.1.1
+region: us-east-1
+```
+
+```yaml {{ label:"nitric.prod.yaml", title: "Google Terraform" }}
+provider: nitric/gcptf@1.1.1
+region: us-east1
+```
+
+```yaml {{ label:"nitric.prod.yaml", title: "Custom" }}
+# Build your own provider to deploy anywhere you like.
+provider: custom/on-prem@1.0.0
+```
+
+</CodeGroup>
+
+We have a few providers built-in with IaC from Pulumi or Terraform, but you can build your own with any tools you prefer and to anywhere you want.
+
+---
+
+## Projects
+
+Projects built with Nitric don't have many restrictions. You can use most languages, most libraries, most tools, most clouds, most services, mostly anything you like. But, you need to have a `nitric.yaml` file in the root of your project.
+
+```yaml {{ label:"nitric.yaml" }}
+name: example
+services:
+    - match: services/*.ts
+      start: npm run dev:services $SERVICE_PATH
+```
+
+Nitric uses this to find your services, then it turns each service into a container, runs them in deployment mode to match the resources you requested and gives the result to the Provider - which either generates IaC (like Terraform) or automatically deploys your app.
+
+So, a project structure might look something like this:
+
+<CodeGroup>
+
+```plaintext {{ title:"JavaScript" }}
+example/
+├── nitric.yaml
+├── services/
+│   └── orders.ts
+│   └── users.ts
+└── package.json
+```
+
+```plaintext {{ title:"Python" }}
+example/
+├── nitric.yaml
+├── services/
+│   └── orders.py
+│   └── users.py
+└── Pipfile
+```
+
+```plaintext {{ title:"Go" }}
+example/
+├── nitric.yaml
+├── services/
+│   ├── orders/
+│   │   └── main.go
+│   └── users/
+│       └── main.go
+├── dockerfile
+└── go.mod
+```
+
+```plaintext {{ title:"Dart" }}
+example/
+├── nitric.yaml
+├── services/
+│   └── orders.dart
+│   └── users.dart
+└── pubspec.yaml
+```
+
+</CodeGroup>
+
+---
+
+## CLI
+
+Nitric has a CLI to help you create, manage, run and deploy your projects. We recommend installing it with a package manager:
+
+<InstallNitric>
+
+```bash {{ title: 'macOS' }}
+brew install nitrictech/tap/nitric
+```
+
+```bash {{ title: 'Windows' }}
+scoop bucket add nitric https://github.com/nitrictech/scoop-bucket.git
+scoop install nitric
+```
+
+```bash {{ title: 'Linux' }}
+curl -L "https://nitric.io/install?version=latest" | bash
+```
+
+</InstallNitric>
+
+### New
+
+You can create a new project from a template with the `new` command.
+
+```bash
+nitric new
+```
+
+### Start
+
+You can run your project locally with the `start` command.
+
+```bash
+nitric start
+```
+
+<Note>
+  Start also emulates the resources you requested, so you can test your app locally. And provides a Dashboard UI to interact with the resources.
```

Comment:
```suggestion
  Start also emulates the resources you requested, so you can test your app locally. It also launches a convenient  Dashboard UI so that you can visualize your application and interact with the resources.
```